user_sync/config/sign_sync.py

```diff
@@ -81,7 +81,10 @@ def __init__(self, args: dict):
         filename, encoding = self._config_file_info()
         self.config_loader = ConfigFileLoader(encoding, self.ROOT_CONFIG_PATH_KEYS, self.SUB_CONFIG_PATH_KEYS)
         self.raw_config = self._load_raw_config(filename, encoding)
-        self._validate(self.raw_config)
+        try:
+            self._validate(self.raw_config)
+        except ConfigValidationError as e:
```

Comment:
Do not re-throw the exception as AssertionException.  Just throw the validation error and catch it in run_sync